debian/kafka-connect-base/Dockerfile

```diff
@@ -48,3 +48,6 @@ VOLUME ["/etc/${COMPONENT}/jars", "/etc/${COMPONENT}/secrets"]
 COPY include/etc/confluent/docker /etc/confluent/docker
 
 CMD ["/etc/confluent/docker/run"]
+
+HEALTHCHECK --interval=5s --timeout=10s --retries=120 \
```

Comment:
maybe we want to use start-period here to work with slower/smaller VMs? since the rest server comes up after  plugins are loaded?

Reply:
Could do, but reading the docs I think `retries` will achieve the same here. We'll poll for a total of ten minutes (5*120) and count as 'unhealthy' after that. WDYT?

Reply:
I suppose that depends on the use case. A longer retry timeout means we'll report errors much later. Also, what errors do we plan to catch once the Rest API comes up though?

Reply:
> Also, what errors do we plan to catch once the Rest API comes up though?

None at all. This healthcheck just ensures that the Kafka Connect REST API is up. It's not comprehensive, but it's 100% more than we currently have :)

Reply:
+1 for `--start-period`.  We know there is a startup delay so I think it makes sense to give the container time to boot without counting the first couple failures "against it" and it makes the actual failure count more accurate.

10 minutes to report unhealthy seems like a long time, although being able to overide in `docker run` and `docker-compose` maybe longer is better?